ucsb-cs156-w24/team03-w24-5pm-2
# Jh menu item create page

In this PR, we implement the UCSBDiningCommonsMenuItem create page, along with its tests.

[Storybook](https://ucsb-cs156-w24.github.io/team03-w24-5pm-2/prs/93/storybook/?path=/story/pages-ucsbdiningcommonsmenuitem-ucsbdiningcommonsmenuitemcreatepage--default)

How to test the create page:
1. Click on UCSBDiningCommonsMenuItem in the navigation bar
2. Create a new item by clicking Create UCSBDiningCommonsMenuItem 
3. Fill out all fields (you won't be able to create without this)
4. After clicking create, your entry should appear, and you should be able to see it after the website automatically takes you back to the index page
5. From here, you can edit/delete your entry

![image](https://github.com/ucsb-cs156-w24/team03-w24-5pm-2/assets/124759864/088d4efc-90ed-4220-9241-032d93fc72c4)

![image](https://github.com/ucsb-cs156-w24/team03-w24-5pm-2/assets/124759864/c773eb83-5702-4b0c-a0ea-4813413c50c4)

Closes #13
